actions/main/src/gradleinttest.sh

```diff
@@ -51,12 +51,17 @@ gradleinttest () {
       echo "Running integration tests against ${IT_JADE_API_URL}"
       cleaniampolicy
     fi
+    if [[ "${test_to_run}" == "check" ]] && [[ "${SONAR_TOKEN}" != "" ]]; then
+      echo "Sonar scan will run after the check action"
+      export sonar_cmd="jacocoTestReport sonar --info"     
```

Comment:
That's true, it's pretty noisy with --info set. I'll turn it off -- we really should notice if it's not working correctly because it should comment the sonar results on the PR.